integtest/spec/spec_helper.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'tmpdir'
+require 'fileutils'
+require 'open3'
+
+ENV['GIT_AUTHOR_NAME'] = 'Test'
+ENV['GIT_AUTHOR_EMAIL'] = 'test@example.com'
+ENV['GIT_COMMITTER_NAME'] = 'Test'
+ENV['GIT_COMMITTER_EMAIL'] = 'test@example.com'
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = '.rspec_status'
+
+  # Disable RSpec exposing methods globally on `Module` and `main`
+  config.disable_monkey_patching!
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
+end
+
+RSpec.shared_context 'tmp dirs' do
+  before(:context) do
+    @tmp = Dir.mktmpdir
```

Comment:
I always favor `let` blocks when creating variables in case some tests don't need the variable and the work to create the variable can be avoided.
Why did you choose to use an instance variable here?

Reply:
`let` variables are scoped to a single example but these live for the entire context. I need these to live for the context because I only want to do each conversion once.